example.html

```diff
@@ -4,6 +4,28 @@
 ---
 
 Clicking on the example will open a static version of the example, powered by <a href="https://nbviewer.jupyter.org/">nbviewer</a>.
+To interact with a "live" version, it is possible from there to download the notebook and run it locally:
+<ul>
+  <li>right-click on the download icon of the notebook (in the upper right corner), which will allow
```

Comment:
What kind of click is it on Mac?

Reply:
right click if one has a mouse or knows how to do this on the touchpad; otherwise ctrl-click ; might also say something like:
> open the context menu on the download (via right-click on Linux and Windows, or ctrl-click on macOS)

Reply:
Ok thanks